src/Command/ExecuteCoreUpgradeScriptsCommand.php

```diff
@@ -108,6 +108,8 @@ protected function execute(InputInterface $input, OutputInterface $output){
 				$fsHelper->move($oldSourcesDir . '/__apps', $oldSourcesDir . '/apps');
 				$output->writeln($plain);
 			} catch (ProcessFailedException $e){
+				$fsHelper->removeIfExists($oldSourcesDir . '/apps');
+				$fsHelper->move($oldSourcesDir . '/__apps', $oldSourcesDir . '/apps');
```

Comment:
Not sure, if that couldn't be executed twice...
Doesn't matter if it is - but better twice than never.